tests/command/cursor-batchsize-001.phpt

```diff
@@ -0,0 +1,86 @@
+--TEST--
+MongoDB\Driver\Command batchSize applies to getMore
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS_CRYPTO(); ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+class Test implements MongoDB\Driver\Monitoring\CommandSubscriber
+{
+    public function executeCommand()
+    {
+        MongoDB\Driver\Monitoring\addSubscriber($this);
+
+        $manager = new MongoDB\Driver\Manager(STANDALONE);
+
+		$bulkWrite = new MongoDB\Driver\BulkWrite;
```

Comment:
Indentation looks off here. PHP source uses four-space indents (not tabs as in C sources). I realize that's confusing :)